views/js/controller/creator/templates/testpart.tpl

```diff
@@ -17,16 +17,6 @@
                             </span>
                             <span class="tlb-end"></span>
                         </span>
-                        <!--<span class="tlb-bar">-->
-                            <!--<span class="tlb-start"></span>-->
-                            <!--<span class="tlb-text-button-box">-->
-                                <!--<a class="tlb-text-button-off section-adder">-->
-                                    <!--<span class="icon-add"></span>-->
-                                    <!--<span class="tlb-text">New Section</span>-->
-                                <!--</a>-->
-                            <!--</span>-->
-                            <!--<span class="tlb-end"></span>-->
-                        <!--</span>-->
```

Comment:
👍

Reply:
We don't have any script for remove comments automatically so... :D